Cabal/doc/nix-local-build.rst

```diff
@@ -118,9 +118,9 @@ across projects. To be more precise:
 1. A **local package** is one that is listed explicitly in the
    ``packages``, ``optional-packages`` or ``extra-packages`` field of a
    project. Usually, these refer to packages whose source code lives
-   directly in a folder in your project (although, you can list an
+   directly in a folder in your project. But you can list an
    arbitrary Hackage package in ``extra-packages`` to force it to be
```

Comment:
extra-packages could be a link (especially important as there it tells that feature is not implemented).

Reply:
Added link to the second occurrence below. But did so blindly (my attempts to build the docs locally with python2 failed :( so please check the syntax.